samples/snippets/fsharp/tour.fs

```diff
@@ -394,7 +394,7 @@ module Arrays =
 
     // You can modify the contents of an an array element by using the left arrow assignment operator.
     //
-    // To learn more about this operator, see: https://docs.microsoft.com/dotnet/articles/fsharp/language-reference/values/index#mutable-variables
+    // To learn more about this operator, see: https://docs.microsoft.com/dotnet/fsharp/language-reference/values/index#mutable-variables
```

Comment:
Same thing here ... should it be ...

*https://docs.microsoft.com/dotnet/fsharp/language-reference/values/#mutable-variables*